code/modules/jobs/job_types/chaplain/chaplain_costumes.dm

```diff
@@ -1,7 +1,16 @@
 //Chaplain Suit Subtypes
 //If any new staple chaplain items get added, put them in these lists
 /obj/item/clothing/suit/chaplainsuit
-	allowed = list(/obj/item/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/cup/glass/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/flashlight/flare/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	allowed = list(
+		/obj/item/book/bible,
+		/obj/item/nullrod,
+		/obj/item/reagent_containers/cup/glass/bottle/holywater,
+		/obj/item/storage/fancy/candle_box,
+		/obj/item/flashlight/flare/candle,
+		/obj/item/tank/internals/emergency_oxygen,
+		/obj/item/tank/internals/plasmaman,
+		/obj/item/gun/ballistic/bow/divine
+	)
```

Comment:
Make this a global list both suits can use (`Initialize() allowed = GLOB.chaplain_suit_items`)